INSTALL.md

```diff
@@ -24,56 +21,142 @@
 ## Installation
 
 ### Deploy the Container
-1. Connect to the RPi, via SSH for example.
-2. 
-    a. Clone the repo.
-
- **OR**
-
-2.
-    b. Create a `tesla_ble_mqtt_docker` folder in your user directory: `cd ~ && mkdir tesla_ble_mqtt_docker && cd $_.`
-
-    Add a docker-compose.yml file: `nano docker-compose.yml`.
-4. Adjust `docker-compose.yml` to your needs. Pay attention to the environment variables :
+There are various methods to deploy the container, I describe the main ones below:
+#### Deploy using Command Line ####
+1. Create a tesla_ble_mqtt_docker folder in your user directory and change directory into it:
    ```yaml
-   environment:
-      - TZ='Europe/London'
-      - TESLA_VIN =<tesla VIN> # Can be found in the App.
-      - MQTT_IP=<IP of the MQTT broker> # In most cases the local IP of HA.
-      - MQTT_PORT=1883 # 1883 is the default port, do not modify unless you specified another.
-      - MQTT_USER=<MQTT username> # Use what's specified in the MQTT broker on HA.
-      - MQTT_PWD=<MQTT password> # Use what's specified in the MQTT broker on HA.
+   cd ~ 
+   mkdir tesla_ble_mqtt_docker 
+   cd tesla_ble_mqtt_docker
    ```
-6. Save and exit.
-7. Create the Docker volume: `docker volume create tesla_ble_mqtt`
-8. Start the container: `docker-compose up -d`
-
-    Check the logs in Portainer (or other). They should look like this:
+2. Download docker-compose.yml and stack.env from the github repository:
+   ```yaml
+   wget https://raw.githubusercontent.com/tesla-local-control/tesla_ble_mqtt_docker/main/docker-compose.yml

```

Comment:
Can we please switch to curl?  Some distributions don't have wget, that's true also for MacOS.

```
curl https://raw.githubusercontent.com/tesla-local-control/tesla_ble_mqtt_docker/main/docker-compose.yml -o docker-compose.yml
```